Sources/content/docs/interfaces.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Interfaces"
+description: Interfaces for the Reactive Machine
+weight: 5
+menu:
+  main: 
+    weight: 20
+---
+
+# Interfaces 
+
+The Reactive Machine interfaces: understanding how to build your application.
+
+* **Orchestrations** describe operations that are composed of one or more other operations. They are written in async/await style, and execute deterministically and reliably. When executing an operation, orchestrations can either perform it (meaning they wait for its completion) or fork it (meaning it executes completely independently of its parent).
+* **Activities** are operations that can be unreliable or nondeterministic, such as calls to external services.
+* *States** represent a small piece of information (cf. key-value pair, or a grain, or virtual actor) that can be atomically accessed via a specified set of read and update operations.
```

Comment:
I think this should be **States**, not *States**

Reply:
That's definitely a typo.

Reply:
Fixed in 9f83499.